config_example.yml

```diff
@@ -0,0 +1,39 @@
+# All the necessary information to deploy to AWS
+aws:
+
+  # The accounts we will be deploying resources to. Each account will have an IAM policy and role created, in addition
+  # to an AWS config rule created in EACH region specified under aws.regions
+  accounts:
+    - account_name: "my-account-1"
+      account_id: "1234567890"
+    - account_name: "my-account-2"
+      account_id: "0987654321"
+
+  # These are the regions that we will be deploying resources in. Every account in aws.accounts will have policies created
+  # in EACH region noted below.
+  regions:
+    - "us-east-1"
+    - "us-east-2"
+    - "us-west-1"
+    - "us-west-2"
+
+  # The role Terraform will be creating, and custodian will be assuming to deploy resources
+  IAM_role_name: "cloud_enforcer_role"
+
+  # The policy that Terraform will be creating, and gives IAM_role_name sufficient privileges
+  IAM_policy_name: "cloud_enforcer_policy"
+
+  # The policy prefix for custodian resources, the suffix will be the resource type, defined in aws.resources (such as s3/ec2/ebs)
+  custodian_policy_prefix: "mandatory_tag_enforcer_"
+
+  # The resources that the custodian policies will be watching, each resource listed below requires its own policy resource
+  # until further progress is made on https://github.com/cloud-custodian/cloud-custodian/issues/1244
+  resources:
+    - "s3"
+
+  # AWS access and secret key used to deploy the IAM roles via Terraform. These credentials must be associated
+  # to a user/role with sufficient privileges to deploy IAM roles defined in the /terraform_templates/ directory
+  provider:
+    region: "us-east-1"
+    shared_credentials_file: "/path/to/.aws/credentials"
```

Comment:
It may be desirable to omit this line from the provider configuration, and letting Terraform pick up AWS credentials using the environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.

This way, the location of the AWS credentials file does not need to be specified in the configuration YML.

Reply:
Had a discussion with Erich Weiler, we ended up choosing to use a credentials file for the time being. I see the value in not specifying this in a file, so this may change in the future, I don't feel strongly either way.

In any case, the `config_example.yml` file should never contain actual private info. The file that will contain account info and any credentials is `config.yml` which is part of `.gitignore`.

Reply:
Probably not a big deal, but this will make the configuration file dependent on the operators AWS setup. Or vise verse.

Reply:
Thought about it some more, decided to remove credentials from the config file, overall its a better approach.

I appreciate the suggestion!